CHANGE_MANAGEMENT.md

```diff
@@ -0,0 +1,148 @@
+# Overview
+## **What is the Digital Quality Implementer's Community?**
+DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality measures. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community. 
```

Comment:
Can we say "focused on Digital Quality Improvement applications"? I think there was agreement that this is broader than just measures

Reply:
```suggestion
DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality improvement applications. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community. 
```

Reply:
Yes that makes a lot of sense.